pom.xml

```diff
@@ -194,7 +204,7 @@
       <plugin>
         <inherited>true</inherited>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.5.1</version>
+        <version>${maven-compiler-plugin.version}</version>
```

Comment:
ps we de-normalized these intentionally to only making properties for versions that are re-used or otherwise require overriding. I think it came from feedback from @michaelsembwever which was that we were making some clutter when blanket version-variabling everything